cli/tsc/dts/lib.deno.unstable.d.ts

```diff
@@ -1475,6 +1475,2211 @@ declare namespace Deno {
       fileName: string,
       source: string,
     ): Diagnostic[];
+
+    /**
+     * Shared properties of an ast node
+     * @category Linter
+     * @experimental
+     */
+    export interface AstBase {
+      /** The byte start and end location of the node */
+      range: [number, number];
+    }
```

Comment:
If you have an `AstBase`, is it possible to get a specific node by checking the `type:` property? For example:

```ts
const node: AstBase = ...;
if (node.kind === "ImportSpecifier") {
  // node is an `ImportSpecifier` here
}
```

I don't think so, right? Is this something a user might want to do?

Reply:
Yes they all have a `type` property. I could type it here as string. That's a good idea.

Reply:
Added it 👍